src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("validUser");
+        user.setAge(18);
+        user.setPassword("password");
+        Storage.people.clear();
+    }
+
+    @Test
+    void registerNullUser_notOk() {
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(null);
+        }, "When User is null, the method should throw RuntimeException");
+    }
+
+    @Test
+    void registerUserWithNullLogin_notOk() {
```

Comment:
Let's make all names of test methods according the common rule from README
Example: methodName_state_expectedBehavior
Each part of name should start with a lowercase letter